flows/actions/add_urn.go

```diff
@@ -40,26 +40,29 @@ func (a *AddURNAction) Validate(assets flows.SessionAssets) error {
 }
 
 // Execute runs the labeling action
-func (a *AddURNAction) Execute(run flows.FlowRun, step flows.Step) ([]flows.Event, error) {
+func (a *AddURNAction) Execute(run flows.FlowRun, step flows.Step, log flows.ActionLog) error {
```

Comment:
So this is less FP... but it does kinda make it easier to ensure you're not forgetting to return some events @nicpottier ?